### ISSUE TITLE
Improve keyboard and screen reader compatibility

### ISSUE THREAD

**Issue by rkingett** [netlify/netlify-cms#4216]
A number of the buttons, check boxes, and edit fields are not accessible to screen readers. For instance, the checkbox that is supposed to enable Markdown in the editor does not have a label attached to it, so the screen reader says, only, check box.

Menus do not gain focus upon opening them with the keyboard, as well.

Steps to reproduce.

1. Download and run NVDA, the free and open source screen reader.
2. Launch the Demo CMS.
3. Navigate via keyboard only. No mouse.

CMS version.

The demo version says 2.0.4.

**Comment by erezrokah** [comment]
Closing in favour of https://github.com/netlify/netlify-cms/issues/1333.
You can list any specific issue in the existing issue
